test/cmdlineTests/metadata_append_standard_json_error/input.json

```diff
@@ -0,0 +1,29 @@
+{
+	"language": "Solidity",
+	"sources":
+	{
+		"A":
+		{
+			"content": "// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\n\ncontract test {}"
+		}
+	},
+	"settings":
+	{
+		"viaIR": true,
+		"optimizer": {
+			"enabled": true
+		},
+		"metadata":
+		{
+			"appendCBOR": false,
+			"bytecodeHash": "ipfs"
```

Comment:
I wanted to say that it would be nice to have a CLI test covering this as well but turns out we do have one, just the naming convention is different so it was not that easy to find. I.e. we have `no_append_incorrect_combination` vs `metadata_append_standard_json_error` here.

I'd rename the 4 test we have covering the new option like this to better fit into our naming scheme and actually use the name of the option we settled on:
- `no_append_metadata` -> `no_cbor_metadata`
- `no_append_incorrect_combination` -> `no_cbor_metadata_with_metadata_hash`
- `metadata_append_standard_json_error` -> `standard_no_append_cbor`
- `metadata_append_standard_json` -> `standard_no_append_cbor_with_metadata_hash`

Reply:
Actually, created a PR myself: #13636.